compiler/rustc_typeck/src/check/fn_ctxt/checks.rs

```diff
@@ -770,55 +770,57 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         let prev_diverges = self.diverges.get();
         let ctxt = BreakableCtxt { coerce: Some(coerce), may_break: false };
 
-        let (ctxt, ()) = self.with_breakable_ctxt(blk.hir_id, ctxt, || {
```

Comment:
This code formatted itself, lol. Let me know if you'd like for me to split this out into a rustfmt commit and a meaningful commit, for readability.

Reply:
Hiding whitespace changes at least makes it reviewable

https://github.com/rust-lang/rust/pull/94248/files?diff=split&w=1